src/ontology/edits/oeo-import-edits.owl

```diff
@@ -35,6 +37,12 @@ Currently, it covers the `ro-extracted.owl`-file which is a subset of the Relati
     
 
 
+    <!-- http://purl.obolibrary.org/obo/IAO_0000116 -->
+
+    <owl:AnnotationProperty rdf:about="http://purl.obolibrary.org/obo/IAO_0000116"/>
```

Comment:
I it is probably not the right line, but I noticed that `editor note` (IAO_0000116) has duplicate labels, one from BFO and one from IAO:
![grafik](https://github.com/OpenEnergyPlatform/ontology/assets/36884905/f7011aed-6226-49a4-922a-f27b90d3e70b)

Reply:
Yes, both contribute these annotation properties (there are several). With removing OMO, there are at least only doubled annotations left. I think, I won't fix that in this PR, unfortunatetly :(

Reply:
From todays oeo-dev-meeting: @areleu opened a feature request for ROBOT some time ago with this topic. https://github.com/ontodev/robot/issues/1099